src/pencilkit.cs

```diff
@@ -246,9 +300,171 @@ interface PKToolPicker {
 		[Export ("colorUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle ColorUserInterfaceStyle { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Create individual instances instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Create individual instances instead.")]
 		[Static]
 		[return: NullAllowed]
 		[Export ("sharedToolPickerForWindow:")]
 		PKToolPicker GetSharedToolPicker (UIWindow window);
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("showsDrawingPolicyControls")]
+		bool ShowsDrawingPolicyControls { get; set; }
+
+		// !missing-null-allowed! 'System.Void PencilKit.PKToolPicker::set_AutosaveName(System.String)' is missing an [NullAllowed] on parameter #0
```

Comment:
How do we handle binding `[NullAllowed]` properties in this case?

Reply:
I believe it should work to add `[NullAllowed]` to the property itself.